lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -1819,9 +1819,13 @@ class SkFontFeature {
 class SkTypeface {}
 
 @JS('window.flutterCanvasKit.Font')
+@staticInterop
 class SkFont {
-  external SkFont(SkTypeface typeface);
-  external Uint8List getGlyphIDs(String text);
+  external factory SkFont(SkTypeface typeface);
+}
+
+extension SkFontExtension on SkFont {
+  external Uint16List getGlyphIDs(String text);
```

Comment:
Huh, I guess we had this typed incorrectly all along? Uint16List seems correct looking at the canvaskit bindings.